lib/overcommit/installer.rb

```diff
@@ -118,6 +125,32 @@ def can_replace_file?(file)
         overcommit_hook?(file)
     end
 
+    def preserve_old_hooks
+      return unless File.directory?(hooks_path)
+
+      ensure_directory(old_hooks_path)
+      Overcommit::Utils.supported_hook_types.each do |hook_type|
```

Comment:
Another (perhaps simpler) option is to move `.git/hooks` to `.git/old-hooks`. This way restoring the old hooks is as simple as doing a `rm -rf .git/hooks` and `mv .git/old-hooks .git/hooks`. 

What do you think?